packages/fluent/scss/chat/_variables.scss

```
@@ -0,0 +1,263 @@
+@use "../core/color-system" as *;
+@use "../core/_variables.scss" as *;
+@use "../toolbar/_variables.scss" as *;
+
+
+/// Chat
+/// @group chat
+/// The box shadow transition of the chat.
+$kendo-chat-transition: var( --kendo-transition, inherit );
+/// Chat
+/// @group chat
+/// The hover text of links in the bubble of the chat.
```

all comments should be in the following format:

```suggestion
/// The hover text of links in the bubble of the chat.
/// @group chat
$kendo-chat-link-hover-text: var( --kendo-link-hover-text, inherit );
/// The text of links in the bubble of the chat.
/// @group chat
$kendo-chat-link-text: var( --kendo-link-text, inherit );
```